styles/themes/base/components/_button-group.scss

```diff
@@ -0,0 +1,53 @@
+// Theme
+// button:                     'ld__button-group-item',
+// buttonActive:               'active',
+// buttonFirst:                'first',
+// buttonLast:                 'last',
+// wrapperDisabled:            'disabled',
+// wrapperRequired:            'required',
+// buttonsWrapper:             'ld__button-group-buttons-wrapper',
+// wrapper:                    'ld__button-group-wrapper',
+
+
+.ld__button-group-wrapper {
+
+}
+
+.ld__button-group-buttons-wrapper {
+  &.required{
+    
+  }
+  &.disabled {
+    filter: opacity(50%);
+
+    .ld__button-group-item {
+      &:hover {
+        background-color: var(--btn-bg-color);
+        border-color: var(--btn-border-color);
+        cursor: not-allowed;
+      }
+      &.active:hover {
+        background-color: var(--btn-group-active-color);
+        border-color: var(--btn-group-active-color);
+      }
+    }
+  }
+}
+
+
+.ld__button-group-item {
+  border-radius: 0;
```

Comment:
Use the [border-start-start-radius](https://drafts.csswg.org/css-logical/#propdef-border-start-start-radius), [border-start-end-radius](https://drafts.csswg.org/css-logical/#propdef-border-start-end-radius), [border-end-start-radius](https://drafts.csswg.org/css-logical/#propdef-border-end-start-radius), [border-end-end-radius](https://drafts.csswg.org/css-logical/#propdef-border-end-end-radius) properties

Reply:
Why?

Reply:
To support languages that are right to left in the future)